src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,25 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double operatingWeight;
+    private boolean isHydraulics;
```

Comment:
Consider using access modifiers for the fields `operatingWeight` and `isHydraulics`. Direct access to class fields is discouraged. Instead, you should encapsulate these fields by making them private and providing public getter methods, which you have already done. This will maintain the integrity of the data.